orga/translations/fr.json

```
@@ -634,6 +634,13 @@
       "no-students-imported-yet": "Dans cet onglet, vous retrouverez les résultats et les attestations de certification des élèves. Vous devez, dans un premier temps, importer la base élèves de votre établissement. ",
       "select-label": "Classe"
     },
+    "join-request-form": {
+      "legal-information": {
+        "hyperlink": "https://pix.fr/dp-formulaire-recuperation-pixorga-sco",
+        "link": "En savoir plus sur mes droits.",
+        "text": "Les informations recueillies sur ce formulaire sont enregistrées dans un fichier informatisé par le GIP Pix pour le compte du ministère de l’éducation nationale en tant que responsable de traitement, pour permettre à votre établissement scolaire d’accéder à son espace Pix Orga en lui envoyant une invitation à rejoindre cet espace."
```

Je suggère les clés suivantes : 

```suggestion
        "more-info-url": "https://pix.fr/dp-formulaire-recuperation-pixorga-sco",
        "more-info-text": "En savoir plus sur mes droits.",
        "text": "Les informations recueillies sur ce formulaire sont enregistrées dans un fichier informatisé par le GIP Pix pour le compte du ministère de l’éducation nationale en tant que responsable de traitement, pour permettre à votre établissement scolaire d’accéder à son espace Pix Orga en lui envoyant une invitation à rejoindre cet espace."
```

---

Je trouve ça lourd le `more-info` devant. `link-text` et `link-url` me semble plus appropriés.

---

À la lecture de ton commentaire je me dis maintenant que `more-info-link-text` et `more-info-link-url` me plairaient encore plus 🤣 mais `link-text` et `link-url` seront déjà très bien :+1:

---

😂 j'ai pris ces noms de clés car elles sont déjà utilisées côté `mon-pix`